networks/suzuka/suzuka-full-node/src/tasks/execute_settle.rs

```diff
@@ -130,17 +131,11 @@ where
 			anyhow::bail!("Invalid DA height: {:?}", da_height);
 		}
 
-		// decompress the block bytes
-		let block = tokio::task::spawn_blocking(move || {
-			let decompressed_block_bytes = zstd::decode_all(&block_bytes[..])?;
-			let block: Block = bcs::from_bytes(&decompressed_block_bytes)?;
-			Ok::<Block, anyhow::Error>(block)
-		})
-		.await??;
+		let block: Block = bcs::from_bytes(&block_bytes[..])?;
```

Comment:
I think the removal of compression should be noted in the description to the PR.

Reply:
It isn't removed it's moved down into the DA, which is noted in the linked discussions. But, I will add.